examples/shader/render_event.rs

```diff
@@ -0,0 +1,90 @@
+//! Simple example demonstrating the use of [`App::init_render_event`] to send events from the
```

Comment:
This should be expanded to say, in effect, "you probably don't want to do this."